Entrega calculo nota clase

juanquintanaing opened the pull request.

juanquintanaing committed:
2ºcommitarchivostxt

juanquintanaing committed:
modificación texto

juanquintanaing committed:
cambio origen a destino

juanquintanaing committed:
ult. modif. txt en destino

juanquintanaing committed:
cambio nombre carpetas y archivos

juanquintanaing committed:
nobre carpeta destino

juanquintanaing committed:
Merge branch 'mmasias:main' into main

juanquintanaing committed:
activida nota final clase

mmasias committed:
Update aDiaDeHoy.md

mmasias commented on entregas/quintanaJuan/src/Java/CalculoNota/CalculoNota.java:
No deje espacios innecesarios

mmasias commented on entregas/quintanaJuan/src/Java/CalculoNota/CalculoNota.java:
examenFinal
examenparcial
evaluacionContinua
evaluacionProfesor

mmasias commented on entregas/quintanaJuan/src/Java/CalculoNota/CalculoNota.java:
examenParcial, perdón!

mmasias commented on entregas/quintanaJuan/src/Java/CalculoNota/CalculoNota.java:
cuide la indentación

mmasias commented:
Atención al detalle...

juanquintanaing committed:
Merge branch 'mmasias:main' into main

mmasias committed:
Update aDiaDeHoy.md

juanquintanaing committed:
Merge branch 'mmasias:main' into main

juanquintanaing committed:
ejercicio Numero Random 2/10

mmasias committed:
Actualización de retos y detalles de entrega

- Eliminado el texto genérico sobre la entrega de retos y sus fechas límite.
- Actualizada la tabla en el README de retos con información específica de entrega para el Reto 001 y el nuevo Reto 002, incluyendo fechas de entrega y ramas específicas.
- Añadido el archivo reto002.md con la estructura inicial y la fecha de entrega especificada.

mmasias committed:
fix: Titulo

mmasias committed:
Dos ejemplos...

mmasias committed:
Tres ejemplos ternarios

mmasias committed:
Enunciado de los ejemplos

mmasias committed:
Enlace en la página de ejercicios

mmasias committed:
Incluye salida esperada en contenido de ejercicios

mmasias committed:
Update ClasificacionEdad.md

juanquintanaing committed:
Merge branch 'mmasias:main' into main

mmasias committed:
Definición de reto 001

mmasias committed:
Refinamos requisito (Human vs Computer)

mmasias committed:
Una imagen...

juanquintanaing committed:
Merge branch 'mmasias:main' into main

juanquintanaing committed:
piedrapapeltijeras

juanquintanaing committed:
asistencia

juanquintanaing committed:
caracol 15/10 clase

Commit:
juego caracol clase 16/10

juanquintanaing committed:
Scanner entrada = new Scanner(System.in);

mmasias committed:
Código!

mmasias committed:
Caracol ternario!

mmasias committed:
Update aDiaDeHoy.md

mmasias committed:
Update README.md

mmasias committed:
Recopilación y ordenado de ejercicios

mmasias committed:
Create .gitignore

mmasias committed:
Ejercicios++

mmasias committed:
Update README.md

mmasias committed:
🪨📄✂️🦎🖖🏻

mmasias committed:
Update aDiaDeHoy.md

juanquintanaing committed:
Merge branch 'mmasias:main' into main

mmasias committed:
Un dia en la vida de Marco
Pendiente:
- La madre
- Los dias

juanquintanaing committed:
Merge branch 'mmasias:main' into main

juanquintanaing committed:
dia de vaije marco, pendiente madre y días

mmasias committed:
Solucion hecha en clase

mmasias committed:
Update ViajeMarco.java

Es más lógico expresarlo así:

seHanEncontrado si la distancia entre Marco y su madre es menor o igual a cero.

mmasias committed:
Propuesta de solución fullTernario 💪🏼💪🏼💪🏼

mmasias committed:
Actualización con soluciones

juanquintanaing committed:
Merge branch 'mmasias:main' into main

mmasias committed:
CarreraCamellos hecho en clase

mmasias committed:
Publicada la solución propuesta de CarreraCamellos

mmasias committed:
Extensión

juanquintanaing committed:
carrera camellos

mmasias committed:
Avance de clase. Inicio de solución de la Batalla

juanquintanaing committed:
Merge branch 'mmasias:main' into main

mmasias committed:
Update README.md

mmasias committed:
Versión minimalista
Vídeo de construcción en https://www.youtube.com/watch?v=bebuAZLYAVE

juanquintanaing committed:
Merge branch 'mmasias:main' into main

juanquintanaing committed:
cambio carrera camellos

mmasias committed:
Update README.md

mmasias committed:
Ejercicio del centro comercial
Además incluyo una corrección en el ejercicio de la carrera de camellos.

mmasias committed:
Solución (hecha en clase) del centro comercial

mmasias committed:
Fix & version ASCII

mmasias committed:
Update README.md

mmasias committed:
Fix, inclusión de hora/minuto y animación

juanquintanaing committed:
Merge branch 'mmasias:main' into main

mmasias committed:
Segundo intento

mmasias committed:
3rd

mmasias committed:
Enlaces

juanquintanaing committed:
Merge branch 'mmasias:main' into main

mmasias committed:
Centro comercial resuelto con métodos estáticos
No he puesto commits individuales :( Disculparme

juanquintanaing committed:
Merge branch 'mmasias:main' into main

juanquintanaing committed:
codigo semi completo, con adivinacion numero random, solo falta las trampas

juanquintanaing committed:
avance en trmapas

juanquintanaing committed:
archivo final dar pista ser caliente

mmasias committed:
eP2

juanquintanaing committed:
Merge branch 'mmasias:main' into main

mmasias committed:
Update aDiaDeHoy.md

mmasias committed:
Ejercicio de temperatura de 7 dias
- Con métodos estáticos
- Adecuado reparto de responsabilidades

mmasias committed:
Refactoriza manejo de temperaturas a matrices

- Cambia variables individuales de temperatura por una matriz.
- Actualiza métodos calcularMinima, calcularMaxima y calcularMedia para usar la nueva matriz.
- Simplifica el código de solicitud de temperaturas con un bucle for.

mmasias committed:
Pequeñas anécdotas acerca de una matriz

mmasias committed:
Cambio extra

juanquintanaing committed:
Merge branch 'mmasias:main' into main

juanquintanaing committed:
cambio

JavierLianoRioz committed:
Update README.md

mmasias committed:
Merge pull request #689 from JavierLianoRioz/patch-1

Modificación del Readme

juanquintanaing committed:
Merge branch 'mmasias:main' into main

mmasias committed:
pyMundo y su solución

mmasias committed:
Update aDiaDeHoy.md

mmasias committed:
Update README.md

mmasias committed:
Update README.md

juanquintanaing committed:
Merge branch 'mmasias:main' into main

mmasias committed:
Update README.md

juanquintanaing committed:
Merge branch 'mmasias:main' into main

juanquintanaing committed:
examen es el archivo del cálculo del º de pregunta del examen

juanquintanaing committed:
array con º carnet

juanquintanaing committed:
suma impares + suma pares

juanquintanaing committed:
calculo º pregunta

juanquintanaing committed:
condicional para modificar calculo de pregunta 3 porque se repite + putprint ºpregunta
